src/test/java/net/amygdalum/tanteemmas/server/SeleniumTest.java

```diff
@@ -0,0 +1,46 @@
+package net.amygdalum.tanteemmas.server;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.concurrent.TimeUnit;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class SeleniumTest {
+
+	private WebDriver driver;
+	private Server server;
+
+	@Before
+	public void setup() {
+		server = new Server();
+		server.startServer();
+		System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver");
```

Comment:
Optional? Because if somebodys has a proper installation, this breaks the test. But I don't know how many folks need this or not.